### ISSUE TITLE
Cannot mine ETH+TON in HiveOS

### ISSUE THREAD

**Issue by Duxter7** [develsoftware/GMinerRelease#637]
The miner says "dual server is not specified" or "dual port is not specified" even though I put TON server and wallet in the Dual Coin field. The same settings work fine in Lolminer, but not in Gminer

**Comment by JohnnyPh** [comment]
You need to fill the pool yourself in miner config and the port for ton is 443

**Comment by Duxter7** [comment]
> You need to fill the pool yourself in miner config and the port for ton is 443

For any ton pool the port is 443? May I where it is mentioned because I couldn't find any info regarding the matter

**Comment by JohnnyPh** [comment]
Yes any ton pool
Yea there is no place mention that but it just work for me haha
But i do encounter another error after every 5 mins "Cuda Error Illegal Instruction Was Encountered"

**Comment by bagyanugraha** [comment]
> 

It doesn't work

**Comment by bagyanugraha** [comment]
stiill cannot  access ton-pool. It says "malperfomed server message"

**Comment by JohnnyPh** [comment]
> stiill cannot access ton-pool. It says "malperfomed server message"

Yup and the dev only increase the fee but doesn’t give a shit to reply any of us
